### DISCUSSION TITLE
TP1 - Ejercicio 7

### DISCUSSION THREAD

**Discussion by Clirussa** [INGCOM-UNRN-P1/cursada-2023#104] [Preguntas y respuestas] [answered]
Hola, les escribimos con @agosgit  porque no entendemos porque la salida es un 11 para cualquier número que ingresamos. 

Esperamos ayuda, 
Muchas gracias

![Ejercicio7-da 11](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/102642611/80151279-2ab5-4f1e-a2a2-7c11c876fe67)

**Comment by martinvilu** [comment, marked as answer]
Hagan cuentas para ese ejercicio (con un lazo), y también las llaves, van siempre.

En especial porque el último if solo agrupa una instrucción, no dos

Ojo que, como decían en clases...

> Lo que pasa en Python queda en Python 

Y operador de comparación es parecido pero no igual al de asignación.
